web/angular-wallet/src/app/layout/components/i18n/i18n.service.ts

```diff
@@ -19,49 +20,45 @@ export class I18nService {
 
 
   constructor(
-      private http: HttpClient,
-      private ref: ApplicationRef,
-      private storeService: StoreService
-    ) {
+    private http: HttpClient,
+    private ref: ApplicationRef,
+    private storeService: StoreService
+  ) {
     this.state = new Subject();
 
-    this.storeService.ready.subscribe((ready) => {
-      if (ready) {
-        this.initLanguage(constants.defaultLanguage || 'en');
```

Comment:
this led to weird overwrite of language settings...

Reply:
nice catch

Reply:
Where does default language get set now?

Reply:
in Settings Entity. It's pulled from constants.js inside the ctor:
https://github.com/burst-apps-team/phoenix/blob/develop/web/angular-wallet/src/app/settings.ts#L41